Providers/Scripts/2.4x-2.5x/Scripts/nxAvailableUpdates.py

```diff
@@ -109,18 +109,21 @@ def GetYumUpdates(Name):
     updates_list = []
     d={}
     # No need to refresh the repo - 'check-update' will do this.
+
+    if os.path.exists('/usr/bin/repoquery'):
```

Comment:
Shouldn't we look for the repoquery installation rather than the particular file?